Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/meals.yml

```diff
@@ -361,3 +361,193 @@
   - type: Tag
     tags:
     - Fruit
+
+- type: entity
+  name: basic noodles
+  parent: FoodBowlBase
+  id: FoodMealBasicNoodles
+  description: A simple ramen bowl of noodles and broth. Yum!
+  components:
+  - type: FlavorProfile
+    flavors:
+      - finenoodles
+      - richbroth
+  - type: Sprite
+    sprite: _NF/Objects/Consumable/Food/bowl.rsi
+    layers:
+    - state: bowl
+    - state: basic-noodles
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 35
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 15
+        - ReagentId: Flavorol
+          Quantity: 10
+
+- type: entity
+  name: miso noodles
+  parent: FoodBowlBase
+  id: FoodMealMisoNoodles
+  description: 
```

Comment:
Missing the description: "Miso, or fermented bean paste, broth with scallion and egg. Robust!" (I think? Am I doing this right, ma?)